### ISSUE TITLE
database file missing

### ISSUE THREAD

**Issue by DarshJoshi6075** [HirushaR/ecom-with-j2ee#4]
bro can you provide database file

**Comment by HirushaR** [comment]
i dont have database right now but database will be create on running. models are created like that.

**Comment by DarshJoshi6075** [comment]
> i dont have database right now but database will be create on running. models are created like that.

yes i created one database and gave connection but it throws error not creating tables on runtime so what to do please help

**Comment by HirushaR** [comment]
in your property file update `spring.jpa.hibernate.ddl-auto=update` to `spring.jpa.hibernate.ddl-auto=create-drop`.
refer https://docs.spring.io/spring-boot/docs/1.1.0.M1/reference/html/howto-database-initialization.html document.

**Comment by DarshJoshi6075** [comment]
Yes it worked but now i am getting different runtime exception like i have
already register customer but i cant login and also getting white label how
to solve this issue please help me i will be very great gesture by you .

**Comment by DarshJoshi6075** [comment]
I cant login getting error even i registered the user and first page is
only sign in no home page is visible can you help me please

**Comment by HirushaR** [comment]
can you send the error or screen shot

**Comment by DarshJoshi6075** [comment]
here is the error i got first i started application on port 8080 then
registered one user  "test" and try to login with the same user still
getting error and i have also added ss of console and one question is in
register form why the 5 fields are given and by default sign in error
appears please help me with this.

**Comment by DarshJoshi6075** [comment]
Pleaseee replyyy i needd youurrr helpp pleaseee.
